router/app/app.go

```diff
@@ -24,52 +24,49 @@ func NewApp(sg *router.RouterImpl) *App {
 	}
 }
 
-func (app *App) ProcPG(ctx context.Context) error {
-	proto, addr := config.RouterConfig().Proto, config.RouterConfig().Addr
-
-	listener, err := reuse.Listen(proto, addr)
+func (app *App) ServeRouter(ctx context.Context) error {
+	address := net.JoinHostPort(config.RouterConfig().Host, config.RouterConfig().RouterPort)
+	listener, err := reuse.Listen("tcp", address)
```

Comment:
we need to respect `proto` setting here

Reply:
tcp includes tcp4 and tcp6, so we decided its fine